token-service/src/main/java/org/p2p/token/service/database/entity/TokenServicePriceEntity.kt

```diff
@@ -0,0 +1,27 @@
+package org.p2p.token.service.database.entity
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import java.math.BigDecimal
+import org.p2p.token.service.model.TokenServiceNetwork
+
+@Entity(
+    tableName = TokenServicePriceEntity.TABLE_NAME
+)
+data class TokenServicePriceEntity(
+    @PrimaryKey
+    @ColumnInfo(name = COLUMN_TOKEN_ADDRESS)
+    val tokenAddress: String,
+    @ColumnInfo(name = COLUMN_TOKEN_PRICE)
+    val price: BigDecimal?,
```

Comment:
if we want to support multicurrency in future, it's better to think about it now

also, i don't think we need to write entity to DB if price is null. 
just check before saving and save only that prices that are not null